PowerShell/JumpCloud Module/Public/Commands/Get-JCCommand.ps1

```diff
@@ -1,10 +1,34 @@
 
 Function Get-JCCommand ()
 {
-    [CmdletBinding(DefaultParameterSetName = 'ReturnAll')]
+    [CmdletBinding(DefaultParameterSetName = 'SearchFilter')]
 
     param
     (
+
+    #### NEW PARAMS
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'The command to execute on the server.')]
+        [String]$command,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Name of the command')]
+        [String]$name,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Command Type')]
+        [string]$commandType,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Launch Type')]
+        [string]$launchType,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Listens To')]
+        [string]$listensTo,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'Schedule')]
+        [string]$schedule,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'The name of the command trigger')]
+        [string]$trigger,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'When the command will repeat')]
+        [string]$scheduleRepeatType,
+        [Parameter( ValueFromPipelineByPropertyName, ParameterSetName = 'SearchFilter', Position = 0, HelpMessage = 'The ID of the organization')]
+        [string]$organization,
```

Comment:
Organization is also something I'm thinking we leave out. I'll explain:

When you set your API Key we check if that key is am MTP key or not. If we just set the env variable for the Org ID. If it is an MTP key we ask the user to select an org to connect to. We won't be able to search different commands by organization with the way the JumpCloud module is currently written. 

I'm in favor of removing this parameter and `ListensTo` just because both will not be very beneficial to us.

Reply:
@kmaranionjc  to remove